pkgs/top-level/python-packages.nix

```diff
@@ -366,6 +366,14 @@ in {
 
   hdmedians = callPackage ../development/python-modules/hdmedians { };
 
+  hoomd-blue = callPackage ../applications/science/molecular-dynamics/hoomd-blue {
```

Comment:
Python packages need to live under `pkgs/development/python-modules`. If the package instead provide as well Python bindings, then the version in `all-packages.nix` needs to be overridden and `toPythonModule` should be used. Using `stdenv.mkDerivation` directly in `python-packages.nix` is not permitted.

Reply:
Fixed. I have used `toPythonModule` and the package has been moved to `python-modules`.